File: Python/amazon-preparation/fizz_buzz.py
def fizz_buzz(n):
    print(f"el pinche numerito: {n}")
    for i in range(1, n + 1):
        if i % 3 == 0 and i % 5 == 0:
            print("Fizz Buzz")
        elif i % 3 == 0:
            print("Fizz")
        elif i % 5 == 0:
            print("Buzz")
        else:
            print(i)

if __name__ == "__main__":

    n = int(input('Incerte su pinche numero: '))

    fizz_buzz(n)

File: Javascript/fizz_buzz.js
const obj = {};
obj["1"] = "Hello";

console.log(obj);

const FizzBuzz = (max) => {
    for (let i = 1; i < max; i++) {
        let out = "";
        if (i % 3 === 0)
            out += "Fizz";
        if (i % 5 === 0)
            out += "Buzz";
        if (i % 7 ===0)
            out += "Bazz";

        console.log(out || i);
    }
};

FizzBuzz(101);